mne/viz/ica.py

```diff
@@ -627,30 +633,48 @@ def plot_ica_scores(ica, scores, exclude=None, labels=None, axhline=None,
     exclude = np.unique(exclude)
     if not isinstance(scores[0], (list, np.ndarray)):
         scores = [scores]
-    n_rows = len(scores)
+    n_scores = len(scores)
+
+    if n_cols is None:
+        # prefer more rows.
```

Comment:
I would have assumed the opposite given that screens are wider than they are tall. Bonus of computing `n_cols` first is that the the `n_rows` can be unified/deindented. WDYT?

Reply:
That is how I originally wrote it as well.

I changed it because:
1. Most common use case is plotting 1 or 2 scores from ECG and EOG artifact rejection, backward compatibility would put these as 1 column.
2. The subplots are wider then they are tall. so it makes sense to add a new row before adding a new column

Reply:
I'm fine with either.

Reply:
Okay I can live with this then!